### ISSUE TITLE
Parser chokes on Gherkin comments

### ISSUE THREAD

**Issue by bencates** [meadsteve/white-bread#21]
Gherkin allows line comments, but white-bread chokes on them when parsing.

Given the file `features/test.feature` with content:

```feature
Feature: Empty file with comment

  Scenario: Empty scenario
    # This line should be ignored
```

White Bread's parser produces the following error

```
10:10:25.934 [debug] Parsing line: "Feature: Empty file with comment"

10:10:25.934 [debug] Parsing line: "Scenario: Empty scenario"

10:10:25.934 [debug] Parsing line: "# This line should be ignored"
** (EXIT from #PID<0.49.0>) an exception was raised:
    ** (CaseClauseError) no case clause matching: "# This line should be ignored"
        lib/white_bread/gherkin/parser/steps.ex:11: WhiteBread.Gherkin.Parser.Steps.string_to_step/1
        lib/white_bread/gherkin/parser/steps.ex:5: WhiteBread.Gherkin.Parser.Steps.add_step_to_scenario/2
        lib/white_bread/gherkin/parser/generic_line.ex:80: WhiteBread.Gherkin.Parser.GenericLine.process_line/2
        (elixir) lib/enum.ex:1261: Enum."-reduce/3-lists^foldl/2-0-"/3
        lib/white_bread/gherkin/parser.ex:27: WhiteBread.Gherkin.Parser.parse_each_line/1
        lib/white_bread/gherkin/parser.ex:11: WhiteBread.Gherkin.Parser.parse_feature/1
        (elixir) lib/task/supervised.ex:74: Task.Supervised.do_apply/2
        (elixir) lib/task/supervised.ex:19: Task.Supervised.async/3
```

**Comment by meadsteve** [comment]
Would you believe I've not got any test cases with a comment in them. I'm taking a look now as it shouldn't be too difficult. Thanks for reporting this :+1:

**Comment by meadsteve** [comment]
Fixed via 58074fdc65bb977af6c828c4d3229eaff958e326

**Comment by meadsteve** [comment]
I'll be releasing this as a version soon.
